src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos/AspireAzureEFCoreCosmosDBExtensions.cs

```diff
@@ -25,7 +25,7 @@ public static class AspireAzureEFCoreCosmosDBExtensions
 
     /// <summary>
     /// Registers the given <see cref="DbContext" /> as a service in the services provided by the <paramref name="builder"/>.
-    /// Configures the connection pooling, logging and telemetry for the <see cref="DbContext" />.
+    /// Enables db context pooling, corresponding health check, logging and telemetry.
```

Comment:
```suggestion
    /// Enables db context pooling, logging and telemetry.
```

There is no health check for cosmos.